docs/src/starknet/101.md

```diff
@@ -0,0 +1,205 @@
+
+After prefunding the address, run:
+sncast account deploy --network sepolia --name my_account
+```
+
+</details>
+
+After creating an account, send enough STRK tokens to the address to cover the price of the
+account creation.
+At least `max_fee` amount of fri (1 STRK = 10<sup>18</sup> fri) should be sent.
+
+> 💡 **Tip**
+> On Sepolia, [this free faucet](https://starknet-faucet.vercel.app/) can be used to fund the account.
+
+
+Then, deploy the account using the command that was provided in output, below `After prefunding the address, run`:
+
+<!-- { "ignored": true } -->
+```shell
+$ sncast account deploy \
+  --network sepolia
+  --name my_account
+```
+
+<details>
+<summary>Output:</summary>
+
+```shell
+command: account deploy
+transaction_hash: 0x[..]
+
+
+To see invocation details, visit:
+transaction: https://sepolia.starkscan.co/tx/0x[..]
+```
+
+</details>
+
+Read more about accounts [here](./account-import.md)
+
+> 💡 **Tip**
+> Existing accounts can be imported to `sncast` as well. See [Importing Accounts](./account-import.md) for more
+> information.
+
+## Calling a Contract
+
+Calling a contract doesn't require an account
+
+```shell
+$ sncast call \
+  --contract-address 0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d \
+  --function name \
+  --network sepolia
+```
+
+<details>
+<summary>Output:</summary>
+
+```shell
+command: call
+response: [0x537461726b6e657420546f6b656e]
+```
+
+</details>
+
+[Read more about calling contracts here](./call.md)
+
+## Sending a Transaction
+
+To send a transaction, invoke a contract.
+
+> 📝 **Note**
+> `--account` argument must be passed before invoke subcommand.
+> This is the same for other commands that send transactions as well.
+
+```shell
+$ sncast \
+  --account my_account \
+  invoke \
+  --contract-address 0x04718f5a0fc34cc1af16a1cdee98ffb20c31f5cd61d6ab07201858f4287c938d \
+  --function transfer \
+  --arguments '0x123, 100' \
+  --network sepolia
+```
+
+> 💡 **Tip**
+> `--arguments` flag supports various kinds of Cairo types including
+> structs `--arguments 'MyStruct { a: 1, b: 2 }` and enums `--arguments MyEnum::Variant1`.
+>
+> See [Calldata Transformation](./calldata-transformation.md) for more details.
+
+<details>
+<summary>Output:</summary>
+
+```shell
+command: invoke
+transaction_hash: 0x[..]
+
+To see invocation details, visit:
+transaction: https://sepolia.starkscan.co/tx/0x[..]
+```
+
+</details>
+
+[Read more about sending transactions here](./invoke.md)
+
+## Declare a Contract
+
+`sncast` uses `scarb` to build contracts and can find contracts by their names (part after `mod` for
+`#[starknet::contract]`).
+To declare a contract, simply pass it name to `sncast`.
+
+> 💡 **Tip**
+> There is no need to run `scarb build` before declaration.
+> `sncast` will do that automatically
+
+Create a project
+
+```shell
+snforge new my_project
+```
+
+From inside the `my_project` directory run
```

Comment:
`````suggestion

```shell
$ snforge new my_project
$ cd my_project
```

Now run:
`````

Reply:
I think it's too "complex" for the purpose of this super simple guide.